lib/http/service.js

```diff
@@ -31,7 +32,17 @@ module.exports = (container) => {
   // automatically parse JSON if it is marked as such. otherwise, just pull the
   // plain-text body contents.
   const bodyParser = require('body-parser');
-  service.use(bodyParser.json({ type: 'application/json', limit: '250kb' }));
+
+  // use a default json limit except for URLs that explicitly need a larger limit.
+  const defaultJsonLimit = bodyParser.json({ type: 'application/json', limit: '250kb' });
+  const largeJsonLimit = bodyParser.json({ type: 'application/json', limit: '100mb' });
+  const largeJsonUrlMatch = match('/:apiVersion/projects/:id/datasets/:name/entities');
+
+  service.use((req, res, next) => {
+    if (largeJsonUrlMatch(req.url))
```

Comment:
I'm noticing that `req.url` will include the query string if there is one, but then `largeJsonUrlMatch()` won't match it:

```js
// Returns an object.
largeJsonUrlMatch('/v1/projects/1/datasets/foo/entities')
// Returns false.
largeJsonUrlMatch('/v1/projects/1/datasets/foo/entities?bar=baz')
```

However, it doesn't really matter now, because the endpoint doesn't use query parameters. I thought I'd mention it, because I could see the endpoint using query parameters in the future. But we could always circle back to it then.

Reply:
I was able to use `req.path` instead of `req.url` to handle this better.